src/controllers/GenericResponse.js

```diff
@@ -0,0 +1,33 @@
+import Api from '../api/Api';
+
+const lookupUrl = `${process.env.GATEWAY_URL}/lookup`;
```

Comment:
Make this as REGISTRY_URL (in our case its same), but we need capability to point our BAP to ONDC registry

Reply:
Done.